docs/source/quantization.rst

```diff
@@ -332,6 +335,95 @@ To learn more about quantization aware training, please see the `QAT
 tutorial
 <https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html>`_.
 
+FX Graph Mode Quantization
```

Comment:
nit: would it make sense to move this to a separate page, and link to it?  The Eager quantization page is already very long.

Reply:
should we create a folder and then split? I think we already have many quantization files under source/